assets/queries/openAPI/2.0/operation_using_password_flow/query.rego

```diff
@@ -0,0 +1,20 @@
+package Cx
+
+import data.generic.openapi as openapi_lib
+
+CxPolicy[result] {
+	doc := input.document[i]
+	openapi_lib.check_openapi(doc) == "2.0"
+
+	doc.paths[p][oper].security[n][key]
+	doc.securityDefinitions[key].flow == "password"
+	doc.securityDefinitions[key].type == "oauth2"
```

Comment:
Based on [this comment](https://github.com/Checkmarx/kics/pull/3664#discussion_r652498677), I think `doc.securityDefinitions[key].type == "oauth2"` is not necessary